packages/coin-kaspa/src/chain-wallet/wallet.ts

```
@@ -85,21 +96,42 @@ export class KaspaChainWallet extends BaseChainWallet {
+      }
+      return balance
     } catch (e) {
       console.error(e);
       return new BigNumber(0);
     }
   }
 
   private async getKrc20Balance(address: string, assetInfo: AssetInfo): Promise<BigNumber> {
+    const tick = assetInfo.contractAddress?.toUpperCase() ?? ''
+    const cache = this.balanceCacheMap[address]
+    if (cache && cache[tick]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)